File: src/main/java/com/atsuishio/superbwarfare/entity/projectile/Sc250Entity.java
package com.atsuishio.superbwarfare.entity.projectile;

import net.minecraft.world.entity.EntityType;
import net.minecraft.world.level.Level;
import software.bernie.geckolib.core.animation.AnimatableManager;

public class Sc250Entity extends Mk82Entity {
    public Sc250Entity(EntityType<? extends Sc250Entity> type, Level level) {
        super(type, level);
        this.noCulling = true;
        this.explosionRadius = 20;
        this.explosionDamage = 500;
    }

    @Override
    public void registerControllers(AnimatableManager.ControllerRegistrar data) {
    }
}

File: src/main/java/com/atsuishio/superbwarfare/entity/projectile/Sc250Entity.kt
package com.atsuishio.superbwarfare.entity.projectile

import com.atsuishio.superbwarfare.resource.BedrockModelLoader
import net.minecraft.resources.ResourceLocation
import net.minecraft.world.entity.EntityType
import net.minecraft.world.level.Level

class Sc250Entity(type: EntityType<out Sc250Entity>, level: Level) : AerialBombEntity(type, level), BasicGeoProjectileEntity {
    override val model: ResourceLocation
        get() = BedrockModelLoader.SC_250_MODEL

    init {
        this.noCulling = true
        this.explosionRadius = 20f
        this.explosionDamage = 500f
    }
}
